Tests/Marketplace/upload_packs.py

```diff
@@ -794,11 +835,12 @@ def main():
     # google cloud bigquery client initialized
     bq_client = init_bigquery_client(service_account)
     packs_statistic_df = get_packs_statistics_dataframe(bq_client)
+    updated_private_packs = []
```

Comment:
```suggestion
    updated_private_packs_ids = []
```
I think its better to change the name here, as the two other pack lists being used in the main function are of type `Pack` or dict

Reply:
Done